modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a subscription
+
+You can create a subscription to connect a service or other event sink to a channel.
```

Comment:
We don't really have a dedicated section about it, but we do explain that it's an event consumer: https://docs.openshift.com/container-platform/4.5/serverless/architecture/serverless-event-architecture.html

It would be nice to have a better section on this, but it probably won't be immediately, so probably not worth blocking the PR on.